.github/workflows/proof_ci.yaml

```diff
@@ -0,0 +1,188 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+# CBMC starter kit 2.9
+name: Run CBMC proofs
+on:
+  push:
+    branches-ignore:
+      - gh-pages
+  pull_request:
+    branches-ignore:
+      - gh-pages
```

Comment:
trivial: to my knowledge, the common-runtime team doesn't have a `gh-pages` branch in any of our repos
but if you just want this here for consistency with how it's set up in other repos, that's fine too

Reply:
We do have a [central definition](https://github.com/model-checking/cbmc-starter-kit/blob/master/src/cbmc_starter_kit/template-for-ci-workflow/proof_ci.yaml) of this file that would be copied into this repository whenever we update the starter kit, so I suggest we keep this here for idempocy